### PR TITLE
Save postmeta attachment image reference on import of products via AP rest.

### PR DESCRIPTION
**Pull request by victorfreitas**
In the creation of the product via API there is no image reference with imported external url.
This facilitates the verification of the image being imported.

### CODE REVIEW

File: includes/api/class-wc-rest-products-controller.php [hunk 1]
```diff
@@ -1078,6 +1078,11 @@ protected function set_product_images( $product, $images ) {
 				if ( ! empty( $image['name'] ) ) {
 					wp_update_post( array( 'ID' => $attachment_id, 'post_title' => $image['name'] ) );
 				}
+
+				// Set for future reference
```
Comment (claudiosanches) [inline includes/api/class-wc-rest-products-controller.php]: Missing a period here.

Comment (victorfreitas) [reply includes/api/class-wc-rest-products-controller.php]: Do you say about the comment?
`Set the image source if present, for future reference.`.

Comment (claudiosanches) [reply includes/api/class-wc-rest-products-controller.php]: Yes, just need to end with a period.
But this new text works great too.

Comment (victorfreitas) [reply includes/api/class-wc-rest-products-controller.php]: Ok, this correction has be resolved.

**Comment by claudiosanches** [comment]
@victorfreitas Thanks.

**Pull request merged**